assets/src/edit-story/components/panels/stylePreset/panel.js

```diff
@@ -119,24 +121,46 @@ function StylePresetPanel() {
     ]
   );
 
+  const extraPropsToAdd = useRef(null);
```

Comment:
Really not a big fun of this. But can't suggest anything better at this point. IMHO, we just need to merge with 65 files and counting and fix forward.

Reply:
Merge what? Not sure what you're referring to here? It's pretty ugly having to do this, but I don't see the consequences that you mention?